src/site/content/en/media/video-and-source-tags/index.md

```diff
@@ -0,0 +1,361 @@
+unsupported types.
+
+{% Aside %} There are a few ways you can dig into the details. Check out [A
+Digital Media Primer for Geeks](//www.xiph.org/video/vid1.shtml) to find out
+more about how video and audio work on the web. You can also use [remote
+debugging](https://developers.google.com/web/tools/chrome-devtools/remote-debugging)
+in DevTools to compare network activity [with type
+attributes](https://googlesamples.github.io/web-fundamentals/fundamentals/media/video-main.html)
+and [without type
+attributes](https://googlesamples.github.io/web-fundamentals/fundamentals/design-and-ux/responsive/notype.html).
+
+Also check the response headers in your browser developer tools to
+[ensure your server reports the right MIME type](//developer.mozilla.org/en/docs/Properly_Configuring_Server_MIME_Types);
+otherwise video source type checks won't work.
+{% endAside %}
+
+### Specify start and end times
+
+Save bandwidth and make your site feel more responsive: use media fragments to
+add start and end times to the video element.
+
+<video controls>
+  <source src="https://storage.googleapis.com/web-dev-assets/video-and-source-tags/chrome.webm#t=5,10" type="video/webm">
+  <source src="https://storage.googleapis.com/web-dev-assets/video-and-source-tags/chrome.mp4#t=5,10" type="video/mp4">
+  <p>This browser does not support the video element.</p>
+</video>
+
+To use a media fragment, add `#t=[start_time][,end_time]` to the media URL. For
+example, to play the video from seconds 5 to 10, specify:
+
+```html
+<source src="video/chrome.webm#t=5,10" type="video/webm">
+```
+
+You can also specify the times in hours:minutes:seconds. For example,
+#t=00:01:05 starts the video at one minute, five seconds. To play only the first
+minute of video, specify #t=,00:01:00
+
+You can use this feature to deliver multiple views on the same video&ndash;like
+cue points in a DVD&ndash;without having to encode and serve multiple files.
+
+For this feature to work, your server must support range requests and that
+capability must be enabled. Most servers enable range requests by default.
+Because some hosting services turn them off, you should confirm that range
+requests are available for using fragments on your site. Fortunately, you can do
+this in your brower developer tools.
+
+After uploading your media and your HTML page to your server, visit the page in
+a browser. Next, open DevTools by right-clicking and selecting **Inspect**.
+Select **Network** in the top menu. From the browser page itself, click reload.
+Back in DevTools, locate your media file in the **Name** column. When you click
+the file, the pages request and response headers should appear. I've shown this
+in the image below. Look for the `Accept-Ranges` header is and verify that is
+says `bytes`. In a the image, I've drawn a red box  around this header. If you
+do not see `butes` as the value, you'll need to contact your hosting provider.
```

Comment:
```suggestion
```
I think we can get rid of this and assume that the audience knows basic Chrome DevTools usage. We should keep the screenshot, though.